test/hotspot/jtreg/runtime/SharedArchiveFile/SharedStrings.java

```diff
@@ -26,17 +26,17 @@
  * @summary Check to make sure that shared strings in the bootstrap CDS archive
  *          are actually shared
  * @requires vm.cds.archived.java.heap
+ * @requires vm.flagless
  * @library /test/lib
- * @modules java.base/jdk.internal.misc
- *          java.management
  * @build SharedStringsWb sun.hotspot.WhiteBox
- * @run driver ClassFileInstaller -jar whitebox.jar sun.hotspot.WhiteBox
+ * @run driver jdk.test.lib.helpers.ClassFileInstaller -jar whitebox.jar sun.hotspot.WhiteBox
  * @run driver SharedStrings
```

Comment:
Includes the changes from [8263549](https://bugs.openjdk.org/browse/JDK-8263549) ( commit [a7aba2b](https://github.com/openjdk/jdk/commit/a7aba2b61cbe937a1aa59636f6e28afdec1e10d0) ) on this file